Packs/FeedZeroDayLiveTIFusion/Integrations/FeedZeroDayLiveTIFUSION/README.md

```diff
@@ -0,0 +1,43 @@
+## Overview
+
+---
+
+Fetch indicators from a ZeroDayLive feed.
+Zero Day Live is our threat intelligence platform. It services multiple security vendors within the industry with the latest intelligence in order to prevent cyber attacks.  
+
+## Configure ZeroDayLive Feed on Cortex XSOAR
+
+---
+
+
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
+2. Search for FeedZeroDayTIFusion.
+3. Click __Add instance__ to create and configure a new integration instance.
+    * __Name__: a textual name for the integration instance.
+    * __URL__: Server URL where the feed is.
+    * __Fetch indicators__: boolean flag. If set to true will fetch indicators.
+    * __Fetch Interval__: Interval of the fetches.
+    * __Reliability__: Reliability of the feed. 
+    * __Traffic Light Protocol Color__: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed. More information about the protocol can be found at <https://us-cert.cisa.gov/tlp>
+    * __Username + Password__ - Credentials to access feeds that require basic authentication. 
+These fields also support the use of API key headers. To use API key headers, specify the header name and value in the following format:
+`_header:<header_name>` in the __Username__ field and the header value in the __Password__ field.
```

Comment:
please add all the other relevant params (Use system proxy settings, Request Timeout...)
refer to the https://github.com/demisto/content/blob/master/Packs/FeedCSV/Integrations/FeedCSV/README.md